versioned_docs/version-2.0.0/users/delegating.md

```diff
@@ -1,80 +1,145 @@
 # Delegating Tokens
 
-A feature of Proof-of-Stake protocols is that token holders can actively participate in the protocol through a mechanism known as **delegating** or **staking** with a validator. CSPR holders can stake their tokens with any validator on a Casper network. Alternatively, it is possible to stake tokens via an exchange or custody provider.
+Casper is a proof-of-stake blockchain, which means CSPR holders can earn rewards by staking their tokens with validators. This process is called delegation, a form of staking, and allows you to participate in securing the network without running a validator node yourself.
```

Comment:
One minor, trivial observation - can the proof-of-stake be capitalised like Proof-Of-Stake. As this is trivial , I've approved the PR.